airgun/widgets.py

```diff
@@ -1718,3 +1718,15 @@ def fill(self, values):
                 widget = get_widget_by_name(self, key)
                 if widget:
                     widget.fill(value)
+
+
+class AutoCompleteSearchInput(TextInput):
```

Comment:
Any other ideas for widget name? It doesn't necessarily mean it's search input, basically any text input with auto complete, smth like 'text input with autocomplete'. `AutoCompleteTextInput` maybe? Or smth similar